index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine">
+  <title>Ayush's Portfolio</title>
+</head>
+<body>
+  <div class="container">
+    <!--Toolbar Begins-->
+    <header class="toolbar">
+      <a href="#" class="logo">Ayush Jha</a>
+      <a href="#" class="menu-icon"><img src="./images/hamburger_icon.svg" alt="menu-icon"></a>
+    </header>
+    <!--Toolbar Ends-->
+    <!--Headline Begins-->
+    <div class="headline-container">
+      <div class="headline-content">
+        <h1 class="headline-welcome">
+          Hey There.<br>
+          I'm Ayush
+        </h1>
+        <h3 class="headline-intro">
+          I'm a Software Developer
+        </h3>
+        <p class="headline-content">
+          I can help you build a product , feature or<br>
+          website Look through some of my work<br>
+          and experience! If you like what you see<br>
+          and have a project you need coded,<br> 
+          don’t hestiate to contact me.
+        </p>
+        <ul class="social-media">
+          <li><a href="#"><img src="./images/Vector-github.svg" alt=""></a></li>
+          <li><a href="#"><img src="./images/Group-linkedin.svg" alt=""></a></li>
+          <li><a href="#"><img src="./images/Vector-angel_list.svg" alt=""></a></li>
+          <li><a href="#"><img src="./images/Vector-twitter.svg" alt=""></a></li>
+          <li><a href="#"><img src="./images/Vector-medium.svg" alt=""></a></li>
+        </ul>
+      </div>
```

Comment:
- [ ] OPTIONAL: It would be better if you expand your headline height to contain the whole page section and stick to the Figma design. Instead of using `height: 550px` use `vh` unit to fill the page.

Check out how it looks:

![image](https://user-images.githubusercontent.com/80612925/176902700-d65f808f-ec98-4247-b635-0f3d0d97ab25.png)

Reply:
Hi, thanks for the suggestion. What vh value should i use? Because if I use 100vh it will stretch for the whole page right? The following sections have different backgrounds so wouldn't that cause problems?